src/Cardano/Wallet/Api.hs

```diff
@@ -25,3 +25,9 @@ type GetWallet = "wallets"
 
 type ListWallets = "wallets"
     :> Get '[JSON] [Wallet]
+
+-- type CreateTransaction = "wallets"
+--     :> Capture "walletId" WalletId
+--     :> "transactions"
+--     :> ReqBody '[JSON] Payment
+--     :> Post '[JSON] Transaction
```

Comment:
Why commenting this out here?

Reply:
There is no `Transaction` type yet - so I just created it as a stub until then. When we merge `Transaction` type I will uncomment it

Reply:
uncommented